htscodecs/rANS_static4x16pr.c

```diff
@@ -1118,7 +1118,7 @@ unsigned char *rans_uncompress_O1_4x16(unsigned char *in, unsigned int in_size,
 	// Build symbols; fixme, do as part of decode, see the _d variant
 	for (j = x = 0; j < 256; j++) {
 	    if (F[j]) {
-		if (x + F[j] > (1<<shift))
+		if (F[j] > (1<<shift) - x)
```

Comment:
Could a similar problem to this also occur [here](https://github.com/samtools/htscodecs/blob/e8d8597e7/htscodecs/rANS_static4x16pr.c#L614)?

Reply:
Yes.  I'll amend that too.